ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -145,6 +145,10 @@ http://www.freedesktop.org/software/systemd/man/systemd.exec.html
 valid for [Service], [Socket], [Mount], or [Swap] sections
 }
 type ${project.artifactId}_unitfile_config_systemd_exec = {
+    'CacheDirectoryMode' ? string # octal notation
+    'CacheDirectory' ? string
```

Comment:
These options take a whitespace-separated list of directory names. The specified directory names must be relative, and may not include "..".